docs/coding-style.md

```diff
@@ -97,18 +99,42 @@ public void MethodB()
 
 ## Naming conventions
 
-Generic words in class names that don't convey meaning (e.g. `Helper`) should be avoided. Overwordy and complex names should be avoided as well.
+### Principles
+
+Keep it minimal and suggestive.
+- Generic words that don't convey meaning (e.g. `Helper`) should be avoided.
+- Overwordy and complex names should be avoided as well.
+- Use positive naming when possible.
+
+### Casing
+
+Protected fields should start with lowercase letter.
+
+### Parameters and variables
 
 Single variable lambdas should use `x` as the variable name (based on lambda calculus λx). Multi variable lambdas should use descriptive names, where `x` can be used for the main iterated item like `(x, index) => ...`. Name `c` can be used for context of Roslyn callback.
 
 Short names can be used as parameter and variable names, namely `SyntaxTree tree`, `SemanticModel model`, `SyntaxNode node` and `CancellationToken cancel`.
 
+### Method names
+
+FIXME Avoid Get prefixes for method names. Save three characters when it only gets x.Foo.Bar.
```

Comment:
I would suggest to vote about this one in the 4th step of our process.

Reply:
The decision/vote was to remove this paragraph.